From 45fb29481636d887ea77d4958cf83293cec7e652 Mon Sep 17 00:00:00 2001
From: Francisco Molina <femolina@uc.cl>
Date: Mon, 26 Apr 2021 12:39:50 +0200
Subject: [PATCH 3/3] src/cbor: patch types

---
 src/cbor.h          | 2 +-
 src/cbor/nanocbor.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/cbor.h b/src/cbor.h
index 54ac33d..8e776f8 100644
--- a/src/cbor.h
+++ b/src/cbor.h
@@ -194,7 +194,7 @@ int8_t cbor_put_int(void *encoder, int value);
  *
  * @return Returns CBOR_SUCCESS on success else CBOR_FAILED.
  */
-int8_t cbor_put_uint(void *encoder, uint value);
+int8_t cbor_put_uint(void *encoder, uint32_t value);
 
 /**
  * @brief Create a CBOR array with a predefined number of elements
diff --git a/src/cbor/nanocbor.c b/src/cbor/nanocbor.c
index e263b8f..b2d44bb 100644
--- a/src/cbor/nanocbor.c
+++ b/src/cbor/nanocbor.c
@@ -233,7 +233,7 @@ int8_t cbor_put_bstr(void *encoder, const uint8_t *bytes, size_t len) {
     }
 }
 
-int8_t cbor_put_uint(void *encoder, uint value) {
+int8_t cbor_put_uint(void *encoder, uint32_t value) {
     if (nanocbor_fmt_uint((nanocbor_encoder_t *) encoder, value) == NANOCBOR_OK) {
         return CBOR_SUCCESS;
     } else {
-- 
2.28.0

