From 57b5ac5561dc454c4d078c409baf2046bc4cf296 Mon Sep 17 00:00:00 2001
From: Lena Boeckmann <einhornhool@mobi31.inet.haw-hamburg.de>
Date: Mon, 25 Nov 2019 14:42:13 +0100
Subject: [PATCH 3/3] adapt cmd-processor.c for custom unit tests

---
 test/cmd-processor.c | 22 ++++++++++++++++++++--
 1 file changed, 20 insertions(+), 2 deletions(-)

diff --git a/test/cmd-processor.c b/test/cmd-processor.c
index 1ce0771..885514c 100644
--- a/test/cmd-processor.c
+++ b/test/cmd-processor.c
@@ -28,7 +28,7 @@
 // Undefine the Unity FAIL macro so it doesn't conflict with the ASF definition
 #undef FAIL
 
-#if !defined(_WIN32) && !defined(__linux__) && !defined(__XC32__) && !defined(__APPLE__) && !defined(ESP32)
+#if !defined(_WIN32) && !defined(__linux__) && !defined(__XC32__) && !defined(__APPLE__) && !defined(ESP32) && !defined(RIOT_APPLICATION)
 #ifdef ATMEL_START
 #include "atmel_start.h"
 #else
@@ -155,6 +155,24 @@ int main(int argc, char* argv[])
 
     return exit_code;
 }
+#elif defined(RIOT_APPLICATION)
+#include <stdio.h>
+#include <stdlib.h>
+#include "cmd-processor.h"
+
+static int exit_code;
+
+static void call_exit(int code)
+{
+    exit_code = code;
+}
+
+
+int run_cmd(const char *command)
+{
+    return parse_cmd(command);
+}
+
 #else
 int processCmd(void)
 {
-- 
2.21.0

