From 8d58f18ad8c1f2cafad443002acfeb1253a541bd Mon Sep 17 00:00:00 2001
From: Benjamin Valentin <benjamin.valentin@ml-pa.com>
Date: Sun, 2 Feb 2020 15:27:26 +0100
Subject: [PATCH] Fix unused parameter warnings

---
 src/ssl.c                 | 4 ++++
 wolfssl/wolfcrypt/types.h | 2 +-
 2 files changed, 5 insertions(+), 1 deletion(-)

diff --git a/src/ssl.c b/src/ssl.c
index fc2aa63bf..3d2d48e58 100644
--- a/src/ssl.c
+++ b/src/ssl.c
@@ -5214,6 +5214,8 @@ static int ProcessBufferTryDecode(WOLFSSL_CTX* ctx, WOLFSSL* ssl, DerBuffer* der
     int* keySz, word32* idx, int* resetSuites, int *rsaKey, int *eccKey,
     int *ed25519Key, void* heap, int devId)
 {
+    (void) heap;
+
     int ret = 0;
 
     if (ctx == NULL && ssl == NULL)
@@ -5341,6 +5343,8 @@ static int ProcessBufferTryDecode(WOLFSSL_CTX* ctx, WOLFSSL* ssl, DerBuffer* der
         XFREE(key, heap, DYNAMIC_TYPE_ECC);
     #endif
     }
+#else
+    (void) devId;
 #endif /* HAVE_ECC */
 #ifdef HAVE_ED25519
     if (ret == 0 && !*rsaKey && !*eccKey) {
diff --git a/wolfssl/wolfcrypt/types.h b/wolfssl/wolfcrypt/types.h
index e80351566..2b9e29dcb 100644
--- a/wolfssl/wolfcrypt/types.h
+++ b/wolfssl/wolfcrypt/types.h
@@ -313,7 +313,7 @@
         #else
         /* just use plain C stdlib stuff if desired */
         #include <stdlib.h>
-        #define XMALLOC(s, h, t)     malloc((s))
+        #define XMALLOC(s, h, t)     malloc((s)); (void)(h); (void)(t)
         #define XFREE(p, h, t)       {void* xp = (p); if((xp)) free((xp));}
         #define XREALLOC(p, n, h, t) realloc((p), (n))
         #endif
-- 
2.20.1

