From c9556f964b916e624b8f960e9c689a7b95626cc0 Mon Sep 17 00:00:00 2001
From: Francisco Molina <femolina@uc.cl>
Date: Mon, 19 Apr 2021 15:50:21 +0200
Subject: [PATCH 4/4] util: fix header inlcudes

---
 util/mem/src/mem.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/util/mem/src/mem.c b/util/mem/src/mem.c
index cd0d59685..213fb26b3 100644
--- a/util/mem/src/mem.c
+++ b/util/mem/src/mem.c
@@ -17,7 +17,8 @@
  * under the License.
  */
 
-#include "os/mynewt.h"
+#include <stdlib.h>
+#include "os/os.h"
 #include "mem/mem.h"
 
 /**
-- 
2.28.0

